src/redux/actions/index.jsx

```diff
@@ -178,3 +189,35 @@ export const sortMoviesError = (error) => ({
 	type: ACTIONS.SORT_MOVIES_ERROR,
 	error,
 });
+
+export const searchMovie = (search) => (dispatch, getState) => {
+	const searchParams = search ? `search=${search}&searchBy=title` : null;
+	const sortParams = `sortOrder=desc&sortBy=${getState().movies.sorting}`;
+	const filterParams =
+		getState().movies.filter === `All`
+			? null
+			: `filter=${getState().movies.filter}`;
+	const paramsString = [searchParams, filterParams, sortParams]
+		.filter((el) => el !== null)
+		.join('&');
```

Comment:
As I see, some logic is duplicated in searchMovie and  sortMovies (like getting filterParams and paramsString), it's quite well sign to move them to separate functions/helpers and reuse.